Content.Server/Paper/PaperSystem.cs

```diff
@@ -34,6 +34,16 @@ public override void Initialize()
             SubscribeLocalEvent<PaperComponent, PaperInputTextMessage>(OnInputTextMessage);
 
             SubscribeLocalEvent<ActivateOnPaperOpenedComponent, PaperWriteEvent>(OnPaperWrite);
+
+            SubscribeLocalEvent<PaperComponent, MapInitEvent>(OnMapInit);
+        }
+
+        private void OnMapInit(EntityUid uid, PaperComponent paperComp, MapInitEvent args)
```

Comment:
doesnt this mean that the book will be mostly empty if you, say, spawn one mid-round
also put an screenshot of an open book in the pr body